config/config.go

```diff
@@ -99,7 +99,10 @@ type Metrics struct {
 
 // OpenTelemetry collector configuration for tracing
 type OtelCollector struct {
-	Protocol string `yaml:"protocol,omitempty"` // http or https or grp
+	CaName     string `yaml:"ca_name,omitempty"`
```

Comment:
```suggestion
	CAName     string `yaml:"ca_name,omitempty"`
```

Reply:
I think we're somewhat inconsistent on casing but these should match the other config options: https://github.com/kiali/kiali/blob/ebbdcfa37e8889c0668c2b1e0d86feb9f846239c/config/config.go#L360-L361